docs/source/conf.py

```diff
@@ -70,7 +70,7 @@ def get_authors_from_citation_file(file_name):
 # ones.
 extensions = [
     "sphinx.ext.autodoc",
-    "sphinx.ext.intersphinx",
+    # "sphinx.ext.intersphinx",
```

Comment:
If it's not needed anymore, why not remove it completely?

Reply:
That should be on and I forgot it in the debugging work. 

I think it is good to leave it on. It allows to cross-link repo documentations. At this point is is only crosslinked with python; later we possible crosslink with the ctapipe or pyeventio docu (I don't see a sphinx doc for pyeventio yet).

Reply:
OK! I didn't know if it's supposed to be on or off but it's good I mentioned either way apparently